README.md

```diff
@@ -52,6 +52,7 @@ This appender uses  [LogzioSender](https://github.com/logzio/logzio-java-sender)
 | Parameter          | Default                              | Explained  |
 | ------------------ | ------------------------------------ | ----- |
 | **inMemoryQueueCapacityBytes**       | *1024 * 1024 * 100*                                | The amount of memory(bytes) we are allowed to use for the memory queue. If the value is -1 the sender will not limit the queue size.|
+| **inMemoryLogsCountLimit**       | *-1*                                | Number of logs we are allowed to have in the queue before dropping logs. If the value is -1 the sender will not limit the number of logs allowed.|
```

Comment:
Limit of bytes you call it capacity - `inMemoryQueueCapacityBytes`
Limit of logs you call it limit. 
I suggest using capacity as well